src/main/java/picard/illumina/CollectIlluminaLaneMetrics.java

```diff
@@ -118,7 +119,7 @@ protected int doWork() {
             }
         }
 
-        IlluminaLaneMetricsCollector.collectLaneMetrics(RUN_DIRECTORY, OUTPUT_DIRECTORY, OUTPUT_PREFIX, laneMetricsFile, phasingMetricsFile, READ_STRUCTURE, FILE_EXTENSION == null ? "" : FILE_EXTENSION);
+        IlluminaLaneMetricsCollector.collectLaneMetrics(RUN_DIRECTORY, OUTPUT_DIRECTORY, OUTPUT_PREFIX, laneMetricsFile, phasingMetricsFile, READ_STRUCTURE, FILE_EXTENSION == null ? "" : FILE_EXTENSION, VALIDATION_STRINGENCY);
```

Comment:
`<rant>` isn't it annoying that one needs to push the validation stringency down this way? it would be so nice if there was an "engine-level" validation stringency that could be "polled" instead of this... `</rant>`